hal/src/b5som/network/quectel_ncp_client.cpp

```diff
@@ -1135,6 +1120,21 @@ int QuectelNcpClient::registerNet() {
     // Ignore response code here
     // CHECK_TRUE(r == AtResponse::OK, SYSTEM_ERROR_UNKNOWN);
 
+    if (ncpId() == PLATFORM_NCP_QUECTEL_BG96) {
+        // FIXME: Force Cat M1-only mode, do we need to do it on Quectel NCP?
+        // Scan LTE only, take effect immediately
+        CHECK_PARSER(parser_.execCommand("AT+QCFG=\"nwscanmode\",3,1"));
+        // Configure Network Category to be Searched under LTE RAT
+        // Only use LTE Cat M1, take effect immediately
+        CHECK_PARSER(parser_.execCommand("AT+QCFG=\"iotopmode\",0,1"));
+
+        // Force eDRX mode to be disabled.
+        CHECK_PARSER(parser_.execCommand("AT+CEDRXS=0"));
+
+        // Disable Power Saving Mode
+        CHECK_PARSER(parser_.execCommand("AT+CPSMS=0"));
```

Comment:
Putting a thought out there that if any of the low power modes were enabled by default on the cell radio (like PSM or eDRX), we are likely waiting too long 3 min before disabling these commands, during which we might have the radio enter PSM or eDRX and give unexpected results. 
I would move these two lines `AT+CPSMS=0` and `AT+CEDRXS=0` to where they were before (right around after init) to be on the safer side and not go into these modes even if they are somehow enabled by default.

Reply:
Sounds reasonable, fixed. 👍